src/client/QXmppSendStanzaParams.cpp

```diff
@@ -18,6 +18,7 @@ class QXmppSendStanzaParamsPrivate : public QSharedData
 {
 public:
     QVector<QString> encryptionJids;
+    QXmppTrustStorage::TrustLevels acceptedTrustLevels;
```

Comment:
```suggestion
    QXmppTrustStorage::TrustLevels acceptedTrustLevels = 0;
```
needs to be initialized